src/components/NewWalletButtons/NewWalletButtons.js

```diff
@@ -1,40 +1,52 @@
 // @flow
 
-import React from 'react'
+import React, { PureComponent } from 'react'
 import { t } from 'ttag'
 
-import WalletFace from 'components/WalletFace'
+import { JLink } from 'components/base'
+
+import {
+  ACTIONS,
+  type WalletAction,
+} from 'pages/WalletsStart/WalletsStart'
+
+import newWalletButtonsStyle from './newWalletButtons.m.scss'
 
 type Props = {|
-  +createWallet: Function,
-  +importWallet: Function,
+  +onClick: (WalletAction) => void,
 |}
 
-const NewWalletButtons = ({
-  createWallet,
-  importWallet,
-}: Props) => (
-  <div className='new-wallet-buttons'>
-    <div className='separator' />
-    <div className='create'>
-      <WalletFace
-        onClick={createWallet}
-        iconName='add'
-        title={t`Create new wallet`}
-        description={t`Create your own wallet to manage on-chain funds`}
-        isTransparent
-      />
-    </div>
-    <div className='import'>
-      <WalletFace
-        onClick={importWallet}
-        iconName='import'
-        title={t`Import wallet`}
-        description={t`Import existing wallet to manage on-chain funds`}
-        isTransparent
-      />
-    </div>
-  </div>
-)
-
-export default NewWalletButtons
+export class NewWalletButtons extends PureComponent<Props> {
+  handleClick = (action: WalletAction) => (event: SyntheticEvent<HTMLDivElement>) => {
+    event.preventDefault()
+
+    this.props.onClick(action)
+  }
+
+  render() {
+    return (
+      <div className={`__new-wallet-buttons ${newWalletButtonsStyle.core}`}>
+        <JLink
```

Comment:
Not in this pull request, but I clearly sure that we need to add into `JRaisedButton` `JLink` modification by `href/to` props and add the new theme.
@edanilov @flexxnn @IViol What do you think about this?